jrpc/internal/service.go

```diff
@@ -232,12 +232,14 @@ func (s *jasperService) ResetTags(ctx context.Context, id *JasperProcessID) (*Op
 
 func (s *jasperService) DownloadMongoDB(ctx context.Context, opts *MongoDBDownloadOptions) (*OperationOutcome, error) {
 	jopts := opts.Export()
-	if err := jopts.BuildOpts.Validate(); err != nil {
-		return &OperationOutcome{Success: false, Text: errors.Wrap(err, "invalid build options").Error()}, err
+	if err := jopts.Validate(); err != nil {
+		err = errors.Wrap(err, "problem validating MongoDB download options")
+		return &OperationOutcome{Success: false, Text: err.Error()}, err
```

Comment:
I would just combine this here and below into one longer line,

Reply:
Done. This is done in several other places. Do you want them all changed like this?